GiniSwitchSDK/Example/GiniTariffSDK/SdkBuilder.swift

```
@@ -30,4 +37,17 @@ struct SdkBuilder {
         
         return sdk
     }
+    
+    static func clientCredentials() -> (clientId:String, clientSecret:String, clientDomain:String) {
+        if let path = Bundle.main.path(forResource: "Credentials", ofType: "plist"),
+            let keys = NSDictionary(contentsOfFile: path),
+            let clientId = keys[self.clientIDKey] as? String,
+            let clientSecret = keys[self.clientSecretKey] as? String,
+            let clientDomain = keys[self.clientDomainKey] as? String {
+            return (clientId, clientSecret, clientDomain)
+        }
+        else {
+            return ("", "", "")
```

I would throw an exception here, since at this point something went wrong or any of the required attributes aren't provided.

---

We don't usually throw exceptions from the SDK. We return errors. So, I'm a bit reluctant to add a random exception here. But I might assert. What do you think?

---

I think assert would be ok then. But yes, when there aren't any exceptions in the SDK it would be wrong to throw one here.